README.md

```diff
@@ -1,2 +1,260 @@
+
+:warning: *When the backend is down (free plan, after 1 hour of inactivity), you need to wait until the Dyno is ready.
+This process shouldn't take longer than 20 seconds.*
+
+The backend of Reliability Score project adds reliability information on top of the iRail API.
+We use 4 years of historical delay information from Infrabel to determine the reliability of a train.
+By performing statistics on the data, we can help commuters to plan their future journeys with the train.
+
+## API
+
+Our API uses the iRail API ([https://docs.irail.be](https://docs.irail.be/)) to provide clients the necessary routing information.
+On top of this data, we use pre-generated JSON files which contain statistical information about a vehicle and the stations of its route.
+
+This information is integrated into the API JSON data and used by our frontend application ([https://github.com/oSoc19/reliability-score-frontend](https://github.com/oSoc19/reliability-score-frontend)) to show how reliable your connection is.
+
+### Resources
+
+We have a single resource available at `/connections?from={departureStation}&to={arrivalStation}&time={time}&date={date}&timesel={arrival or departure}`.
+In comparison to the iRail API documentation, all of the arguments are required. In case an argument is missing, a `HTTP 400: Bad request` is returned.
+
+- **from**: The name of the departure station, for example: `Vilvoorde`.
+- **to**: The name of the arrival station, for example: `Brugge`.
+- **time**: The time of the arrival or departure, for example: `0945` for 9h45 in the morning.
+- **date**: The date of the arrival or departure, for example: `220719` for 22 Jul 2019.
+- **timesel**: Use the provided `time` and `date` parameters as *to arrive at* or *to depart from*. The possible values are: `departure` or `arrival`.
+
+Besides the iRail API data (see the documentation of the iRail API for more information), we added 2 properties to the response:
+
+- **reliability_graph**: Contains the frequency of the delays for a vehicle at a certain station. The graph contains 16 buckets:
+    - **Bucket 0**: Negative delays (trains that departed earlier than planned).
+    - **Bucket 1**: On time or a delay < 1 min.
+    - **Bucket 2-15**: Bucket number indicates the border, for example bucket 7: all delays between 6 and 7 min.
+    - **Bucket 16**: > 15 min. delay
+- **reliability_transfer_time**: Uses the highest probable delays of a via connection to calculate the time (in seconds) a commuter has to transfer between 2 trains at the via station. It's comparable to the `timeBetween` property of the iRail API, except that we use the statistical information to find out if there's a big chance to miss a transfer.
+
+### Example
+
+**Request**: [https://reliability-score.herokuapp.com/connections?from=Diegem&to=Brugge&time=0851&date=220719&timesel=departure](https://reliability-score.herokuapp.com/connections?from=Diegem&to=Brugge&time=0851&date=220719&timesel=departure)
+
+**Response**:
```

Comment:
Note which data is part of Reliability Score and which is part of the regular iRail API